src/pages/migrating/guide/design.mdx

```diff
@@ -72,55 +73,160 @@ Beta kit is available in the White and Gray 100 themes.
 
 ## Components
 
+### Notifications
+
+Toast notifications have been refactored to now allow interactive content in
+them, like links and buttons. To support the usecase of having a toast with or
+without interactive content, we have introduced two types of toast
+notifications—polite and assertive.
+
+See Carbon's
+[toast notification](https://v11.carbondesignsystem.com/components/notification/usage/#toast-notifications)
+usage guidance for more information.
+
+<Row>
+<Column colLg={8}>
+
+![Example of toast notifications stacking.](images/05_toast_context_608.gif)
+
+</Column>
+</Row>
+
+### Popover
```

Comment:
@joshblack are these components in good enough shape that we should be giving guidance on their usage?

Reply:
@sstrubberg I think we can for the beta if people understand that this stuff is subject to change.